boards/espressif/esp32_devkitc/doc/index.rst

```diff
@@ -8,7 +8,7 @@ with integrated Wi-Fi & dual-mode Bluetooth. The ESP32 series employs a
 Tensilica Xtensa LX6 microprocessor in both dual-core and single-core
 variations. ESP32 is created and developed by Espressif Systems, a
 Shanghai-based Chinese company, and is manufactured by TSMC using their 40nm
-process. For more information, check `ESP32-DevKitC-WROVER`_.
+process. For more information, check `ESP32-DevKitC`_.
```

Comment:
Need to update L252 accordingly

Reply:
thx, the link target needs to be updated too.